src/viz-population-over-time/VizPopulationOverTime.js

```diff
@@ -13,7 +14,10 @@ import { formatAsNumber, getDataWithPct, highlightFade } from "../utils";
 import ColorLegend from "../color-legend";
 import { CUSTOM_ID } from "../controls/TwoYearRangeControl";
 
+const CHART_HEIGHT = 430;
```

Comment:
These values don't peg us to specific screen resolutions, right?

Reply:
well, to a degree ... the width flexes with the viewport but the height is fixed. for some charts we specify a different mobile height but I'm not sure whether that's necessary here; because the areas are stacked, the height helps with readability even if it does create kind of a squarish aspect ratio. Only on a phone turned to landscape mode should you be really unable to fit the whole thing on your screen.